icaruscode/Timing/PMTTimingCorrectionsProvider.h

```diff
@@ -19,32 +19,24 @@
 #include "fhiclcpp/types/Sequence.h"
 #include "cetlib_except/exception.h"
 
-#include "lardata/DetectorInfoServices/DetectorClocksService.h"
-
+// Local
 #include "icaruscode/Timing/PMTTimingCorrections.h"
 
 // Database interface helpers
-#include "wda.h"
+#include "larevt/CalibrationDBI/Providers/DBFolder.h"
 
 // C/C++ standard libraries
-#include <memory> // std::unique_ptr<>
-#include <optional>
 #include <string>
```

Comment:
We also want, and always did,
```cpp
#include <map>
```
And to be pedantic, which I always am, also `<cstdint>` for `uint64_t` and similar. Although to be even more pedantic, it would be C header, `<stdint.h>`, since we are not using the namespace (`std::uint64_t`).